bugsnag-kmp/src/jsTest/kotlin/com/bugsnag/kmp/EventTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.bugsnag.kmp
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+internal class EventTest {
+
+    @Test
+    fun eventTest() {
+        val jsEvent = JsEvent("Error", "An error occurred")
+        val testEvent = Event(jsEvent)
+
+        testEvent.apiKey = API_KEY
+        testEvent.context = CONTEXT
+        testEvent.groupingHash = "new groupingHash"
+        testEvent.severity = Severity.WARNING
+        testEvent.user = User(id = USER_ID, email = USER_EMAIL, name = USER_NAME)
+        testEvent.addMetadata("new_1", "scenario", "EventScenario 1")
+        testEvent.addMetadata("new_2", "scenario", mapOf("test2" to "EventScenario 2"))
```

Comment:
this should populate the map with more types, including numbers, booleans, lists and maps